packages/fxa-settings/src/components/CardHeader/index.stories.tsx

```diff
@@ -39,6 +44,11 @@ export const BasicWithCustomSubheading = storyWithProps({
   subheadingText: MOCK_SUBHEADING,
```

Comment:
Hmm I'm not sure about `headingAndSubheadingFtlId`, that one's for `BasicWithSubheading`, not `BassicWithCustomSubheading`.

> Should this story be named CustomHeadingAndCustomSubheading?

I hesitate because right now it corresponds with the code. I could make the story `'Heading and custom subheading (basic with custom subheading)'` but I'm not sure that's more clear. I'm not sure about "CustomHeading" since every component uses a custom heading 🤔